src/components/Dasboard/Cards/DasboardCards.tsx

```diff
@@ -23,15 +78,185 @@ export default function DashboardCard({
 			marginTop: theme.spacing(0),
 		},
 	}));
+	const [currentFilter, setCurrentFilter] = useState<any>();
+
+	/*setting first filter as default filter*/
+	useEffect(() => {
+		if (cardFilter && cardFilter.length) {
```

Comment:
`cardFilter && cardFilter.length` could be simplified to `cardFilter?.length`

Reply:
Ohky